examples/err_cmd/README.md

```diff
@@ -0,0 +1,147 @@
+`file2`. If it does, the code marks the datum as recovered, and the
+job succeeds. If it does not, the job fails.
+
+Create a pipeline by running the following command from the `examples/err_cmd/`
+directory:
+
+```bash
+$ pachctl create pipeline -f error_test.json
+```
+
+Verify that the pipeline was successfully created:
+
+```bash
+$ pachctl list pipeline
+NAME       VERSION INPUT    CREATED       STATE / LAST JOB
+error_test 1       input:/* 5 seconds ago running / starting
+```
+
+## Add Files to the Input Repository
+
+Now, let's add some files to the input repository to watch how your pipeline
+code and error code work.
+
+You will add three files, `file1`, `file2`, and `file3`, that each contains one
+line in them.
+
+1. Add `file1`:
+
+   ```bash
+   $ echo "foo" | pachctl put file input@master:file1
+   ```
+
+   When you add `file1`, your pipeline should succeed:
+
+   ```bash
+   $ pachctl list job --no-pager
+   ID                               PIPELINE   STARTED        DURATION           RESTART PROGRESS  DL UL STATE
+   c8860dae5a054ec38a33068f75fe9690 error_test 13 seconds ago Less than a second 0       1 + 0 / 1 4B 4B success
+   ```
+
+   As you can see in the `PROGRESS` column – `1 + 0 / 1`, you have one
+   successfully processed datum.
+
+1. Add `file2`:
+
+   ```bash
+   $ echo "bar" | pachctl put file input@master:file2
+   ```
+
+   Processing of this datum will succeed as well, but the datum will be marked
+   as *recovered* because it does not contain `file1` as per our pipeline code:
+
+   ```bash
+   $ pachctl list job --no-pager
+   ID                               PIPELINE   STARTED       DURATION           RESTART PROGRESS      DL UL STATE
+   bc3da288ff884d5a9bcb312dd6cf07cb error_test 3 seconds ago Less than a second 0       0 + 1 + 1 / 2 0B 0B success
+   c8860dae5a054ec38a33068f75fe9690 error_test 3 minutes ago Less than a second 0       1 + 0 / 1     4B 4B success
+   ```
+
+   In the `PROGRESS` column, you can see that the last job did not have
+   any successfully processed datums, but it had a skipped datum and a
+   recovered datum – `0 + 1 + 1 / 2`.
+
+1. Add `file3`:
+
+
+   ```bash
+   $ echo "baz" | pachctl put file input@master:file3
+   ```
+
+   Because the processed datum does not have nor `file1`, neither
+   `file2`, this job results in failure. Therefore, both `cmd`
+   and `err_cmd` codes result in non-zero status:
+
+   ID                               PIPELINE   STARTED        DURATION           RESTART PROGRESS      DL UL STATE
+   272370ec03c24cc1be660bf97403712f error_test 26 seconds ago Less than a second 0       0 + 2 / 3     0B 0B failure: failed to process datum:...
+   bc3da288ff884d5a9bcb312dd6cf07cb error_test 6 minutes ago  Less than a second 0       0 + 1 + 1 / 2 0B 0B success
+   c8860dae5a054ec38a33068f75fe9690 error_test 10 minutes ago Less than a second 0       1 + 0 / 1     4B 4B success
+
+   In the last pipeline run, you can see that two datums were skipped.
```

Comment:
Remove this sentence. This is a bug that adele is fixing. We are not supposed to skip recovered datums, we are supposed to retry them.